src/back/TicketManager/src/Core/TicketManager.Core.Services/Extensions/JsonConverters/DateTimeConverter.cs

```diff
@@ -0,0 +1,19 @@
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace TicketManager.Core.Services.Extensions.JsonConverters;
+
+public class DateTimeConverter : JsonConverter<DateTime>
```

Comment:
Maybe we start testing converters

Reply:
Yea, that would be nice. I add issue to jira in case there are some technical issues with it.